src/catalog/src/durable/objects.rs

```diff
@@ -1150,6 +1150,11 @@ impl ItemValue {
                 assert_eq!(tokens.next(), Some("VIEW"));
                 CatalogItemType::MaterializedView
             }
+            Some("CONTINUAL") => {
+                assert_eq!(tokens.next(), Some("TASK"));
+                // TODO(ct): CatalogItemType::ContinualTask
+                CatalogItemType::MaterializedView
+            }
```

Comment:
I know it's a decent amount of boilerplate, but I really would prefer a `CatalogItemType::ContinualTask`. No need to do it in this PR but it would be great as a follow up!